lib/onelogin/ruby-saml/settings.rb

```diff
@@ -258,6 +259,7 @@ def get_sp_key
         :idp_cert_fingerprint_algorithm            => XMLSecurity::Document::SHA1,
         :compress_request                          => true,
         :compress_response                         => true,
+        :message_max_bytesize                      => 250000,
```

Comment:
Can this default value be read from SamlMessage.MAX_BYTE_SIZE?

Reply:
@pitbulk ha, I missed that! Just pushed a commit that removes it 👍